payments_service/braintree/views.py

```diff
@@ -36,52 +37,53 @@ def post(self, request):
         if not form.is_valid():
             return error_400(response=form.errors)
 
-        buyer_uuid = request.user.uuid
+        transaction = Transaction(request.session)
+        transaction.create(request.user, form.cleaned_data['plan_id'])
+
         try:
-            self.set_up_customer(buyer_uuid)
+            self.set_up_customer(request.user)
             pay_method_uri = self.get_pay_method(
-                buyer_uuid,
+                request.user,
```

Comment:
could this just be ``request.user.pk`` ? It looks like the ``__str__()`` for that is buried in Django, not sure if we want to rely on that.